app/src/main/java/RetrofitClientInstance.kt

```diff
@@ -1,20 +0,0 @@
-import retrofit2.Retrofit
-import retrofit2.converter.gson.GsonConverterFactory
-
-object RetrofitClientInstance {
-
-    private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://weather.com/swagger-docs/ui/sun/v1/sunV1HourlyForecast.json"
```

Comment:
Rationale: File was redundant as there was another retrofit client instance with the same base_url. Projects shouldn't have redundant files when it can be avoided.